website/content/en/blog/product-thinking-for-platforms.md

```diff
@@ -0,0 +1,134 @@
+
+### Value for customers
+
+Customers? What customers?
+
+When you solve a problem for a user, you create value for them. This is at the core of product thinking and it's why great product teams obsess over their customers. If you take the time to learn as much as possible about what your users are trying to accomplish and how they go about it, your problem of figuring out what to build goes away.
+
+You could study your users from afar and try to intuit what their needs are, but that's entirely unnecessary. You can have all of the insight you need directly from users. Go out and talk to them! It's good to stay focused and stick to discovering problems rather than solutioning (users are just as guilty of solutioning for their own problems). The first step is to create the room to learn. By stepping back and listening, you develop empathy and see what your users are facing. With this approach, you have lots of valuable information on real problems and can move confidently to creating solutions. These solutions can be smaller and more focused, creating faster feedback loops.
+
+
+### Continuous Approach
+
+The Product Mindset differs from the traditional project-oriented management approach. While a project is usually focused around larger phases of requirements gathering, implementation and deployment, product thinking is meant to operate continuously. When using product thinking, the goal is to stay close to the customer, figure out their needs and deliver value to them in a continuous cycle. By engaging with customers continuously you significantly raise the odds of delivering value for them. And when you build a track record of delivering value, you build trust with customers and they keep coming back to you with new opportunities. Contrast this with the project oriented approach where a large project that take huge amounts of time and money to implement misses the mark with the underlying need. In this case everyone usually wants to forget this happened and you end up with another partially implemented solution that becomes a drag.
+
+This mindset shift from project to product is essential for creating successful platforms.
+
+
+## How does Product Thinking fit with platforms?
+
+Modern platforms and platform engineering are focused on reducing cognitive load for engineers and allowing them to focus on creating business value. It only makes sense to treat the platform as a product that becomes part of the value chain of the business. 
+
+Platforms are for building things and running them. A platform user needs to be able to do both.
+
+
+### Find the platform user and value
+
+If we accept this premise (and we'll get into why you would want to below) then we need to figure out how to create value for our users. As we mentioned above, our users are creating value for the business and anything we can do to aid in that process will be of value to them.
+
+This is where the rubber hits the road with product thinking and platform engineering. 
```

Comment:
hmm okay if we take Product Thinking as a name and have it camel case everywhere then maybe platform engineering as well? Or is this now a good reason to stick with lower case? :D Anyways: one Style would be great.

Reply:
platform engineering seems to be lower case in the examples I find on Gartner or platformengineering.org